include/dataformats/FragmentHeader.hpp

```diff
@@ -138,6 +150,57 @@ enum class FragmentErrorBits : size_t
   kInvalid = 32       ///< Error bit 32 and higher are not valid (error_bits is only 32 bits)
 };
 
+/**
+ * @brief This enumeration should list all defined Fragment types
+ */
+enum class FragmentType : fragment_type_t
+{
+  kFakeData = 0,   ///< Data created in dfmodules' FakeDataProducer
+  kReplayData = 1, ///< Data created from frame archive in readout
+  kEmuFEMB = 2,    ///< Data created in WIB's FEMB emulator
+  kUnknown         ///< Used when given a string that does not match any in s_fragment_type_names
```

Comment:
Would it make sense to have `kUnknown = max value of fragment_type_t` so its value doesn't change when we add new fragment types?